src/pages/LEVANTE/UserSurvey.vue

```diff
@@ -87,6 +89,23 @@ const showAndPlaceAudioButton = (playAudioButton, el) => {
   }
 };
 
+function saveSurveyData(survey) {
+  const data = survey.data;
+  data.pageNo = survey.currentPageNo;
+  window.localStorage.setItem(STORAGE_ITEM_KEY, JSON.stringify(data));
+}
+
+function restoreSurveyData(survey) {
```

Comment:
This will work as long as they restart from the same device. Do we expect them to start on a different device. Also, if they do that will cause duplicate entries since same result might be posted twice. Maybe not relevant here though.